### DISCUSSION TITLE
下载doris-2.0.2-rc05后，使用 build.sh 编译报错

### DISCUSSION THREAD

**Discussion by zhaofei1193** [apache/doris#27249] [RFC]
执行 .build.sh报错：

Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
libtool is found
Strip thirdparty libraries
===== begin build unixODBC-2.3.7
checking for a BSD-compatible install... /bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for gawk... (cached) gawk
checking for bison... no
checking for byacc... byacc
checking for style of include used by make... GNU
checking for gcc... /bin/../bin/clang
checking whether the C compiler works... no
configure: error: in `/workspace/open-source/doris-2.0.2-rc05/thirdparty/src/unixODBC-2.3.7':
configure: error: C compiler cannot create executables
See `config.log' for more details

**config.log中错误如下：**

configure:3704: /bin/../bin/clang -V >&5
clang-11: error: argument to '-V' is missing (expected 1 value)
clang-11: error: no input files
configure:3715: $? = 1
configure:3704: /bin/../bin/clang -qversion >&5
clang-11: error: unknown argument '-qversion'; did you mean '--version'?
clang-11: error: no input files
configure:3715: $? = 1
configure:3735: checking whether the C compiler works
configure:3757: /bin/../bin/clang -I/home/dol/workspace/open-source/doris-2.0.2-rc05/thirdparty/installed/include -Wno-int-conversion -Wno-implicit-function-declaration  -L/home/dol/workspace/open-source/doris-2.0.2-rc05/thirdparty/installed/lib conftest.c  >&5
/bin/../bin/ld: cannot find crtbegin.o: No such file or directory
/bin/../bin/ld: cannot find -lgcc
/bin/../bin/ld: cannot find -lgcc_s
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
configure:3761: $? = 1
configure:3799: result: no
configure: failed program was:

**Comment by ixzc** [comment]
Which system？
you can refer this:https://doris.apache.org/zh-CN/docs/dev/install/source-install/compilation-general

**Comment by AcKing-Sam** [comment]
Hi, I have encountered the same question in Ubuntu 22.04.
You can check the file `doris/thirdparty/src/unixODBC-2.3.7/config.log` to see the detailed problem.
I believe in this way you can fix it!
